src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HttpHealthcareApiClient.java

```diff
@@ -197,6 +198,70 @@ public Empty deleteFhirStore(String name) throws IOException {
     return client.projects().locations().datasets().fhirStores().delete(name).execute();
   }
 
+  @Override
+  public String retrieveDicomStudyMetadata(String dicomWebPath) throws IOException {
+    String[] webPathSplit;
```

Comment:
nit: we should probably create a utility class for handling path parsing, since it can be useful for other modalities as well.